grpc-ballerina/src/grpc/tests/02_array_field_type_client.bal

```diff
@@ -19,7 +19,7 @@ import ballerina/test;
 
 final HelloWorld2BlockingClient HelloWorld2BlockingEp = new ("http://localhost:9092");
 
-@test:Config {}
+@test:Config {enable:true}
```

Comment:
Do we need to specifically enable these tests?

Reply:
No, we don't need it. I add this because it is easy to disable if needed.

Reply:
How about doing it at the Gradle level(`-x test`) or perhaps grouping the tests?.

Reply:
grouping the tests would be a better option. I removed it, for now, will check for proper grouping